tests/src/test/scala/com/nvidia/spark/rapids/GpuMultiFileReaderSuite.scala

```diff
@@ -74,4 +74,21 @@ class GpuMultiFileReaderSuite extends AnyFunSuite {
       assertResult(false)(multiFileReader.next())
     }
   }
+
+  test("test column size exceeding cudf limit") {
```

Comment:
The test isn't asserting anything

Reply:
Currently this test is WIP to replicate the scenario in my development machine.